substra/sdk/client.py

```diff
@@ -369,6 +378,17 @@ def login(self, username, password):
         self._token = self._backend.login(username, password)
         return self._token
 
+    @logit
+    def logout(self) -> None:
+        """
+        Log out from a remote server, if Client.login was used
+        (otherwise, nothing happens)
+        """
+        if not self._backend:
+            raise exceptions.SDKException("No backend found")
+        if hasattr(self._backend, "logout"):
+            self._backend.logout()
```

Comment:
I would check the client type rather than `hasattr` if the purpose of this is to check that we have a remote client.

Reply:
That makes sense, but isn't the purpose at this level just to check whether the client has a `logout` function, regardless of what kind of client it is?

Reply:
Regarding my main comment, I think it would make more sense that all client have a logout, as they all have a login method